src/main/java/kr/codesqaud/cafe/controller/ArticleController.java

```diff
@@ -20,14 +20,20 @@ public class ArticleController {
 
     public ArticleController(ArticleRepository articleRepository) {
         this.articleRepository = articleRepository;
+    }
 
+    @GetMapping("/createForm")
+    public String create(@SessionAttribute(name = ConstConfig.SESSION_ID) User user, Model model) {
```

Comment:
```suggestion
    public String create(@SessionAttribute(name = ConstantConfig.SESSION_ID) User user, Model model) {
```

가능하면 줄임말보다는 원래의 단어가 좋다고 생각합니다.